reframe/core/schedulers/slurm.py

```diff
@@ -290,10 +296,12 @@ def _set_nodelist(self, job, nodespec):
     def _update_state(self, job):
         '''Check the status of the job.'''
 
-        completed = _run_strict(
-            'sacct -S %s -P -j %s -o jobid,state,exitcode,nodelist' %
-            (datetime.now().strftime('%F'), job.jobid)
-        )
+        with env.modify_env(variables={'SLURM_TIME_FORMAT': 'standard'}):
```

Comment:
Passing a dictionary would work, even with my context manager, but if you want to guarantee the order of the variables (here not the case, though), you should pass it as a list of tuples:

```python
[('SLURM_TIME_FORMAT', 'standard')]
```